en/Extensions/Plugin security.md

```diff
@@ -0,0 +1,39 @@
+The Obsidian team takes security seriously. This page explains the risks involved when installing community plugins, and what the Obsidian team does to address them.
+
+> [!tip]
+> If you're working with sensitive data and wish to install a community plugin, we recommend that you perform an independent security audit before using it.
+
+## Restricted mode
+
+By default, Obsidian runs in Restricted Mode to prevent third-party code execution. Only disable Restricted mode if you trust the authors of the plugins that you install.
+
+To turn off Restricted mode:
+
+1. Open **Settings**.
+1. In the side menu, select **Community plugins**.
+1. Select **Turn on community plugins**.
+
+To turn on Restricted mode:
+
+1. Open **Settings**.
+1. In the side menu, select **Community plugins**.
+1. Next to **Restricted mode**, select **Turn on**.
+
+Installed plugins remain in your vault even if you turn on Restricted mode, but are ignored by Obsidian.
+
+## Plugin capabilities
+
+Due to technical reasons, Obsidian is unable to restrict plugins to specific permissions or access levels. As a result, consider the following examples of what community plugins can do:
```

Comment:
Another big reason is that not restricting plugins provide a lot more possibility than a strict API, we might want to add this as a positive note

Reply:
I don't think it's the right place for it. The target audience for this document is an end user who's assessing the risk of using plugins. We could reframe "Obsidian is unable to restrict" to "Obsidian doesn't restrict", to signal it being a conscious decision, but I think this page should give the hard truth without any sugar coat.

Reply:
I changed it to "Obsidian is unable to reliably restrict", but I think "Obsidian doesn't restrict" would work too. Let me know if you prefer that more.